UPGRADE.rst

```diff
@@ -49,6 +49,42 @@ returned by the Client-Server API:
     # configured on port 443.
     curl -kv https://<host.name>/_matrix/client/versions 2>&1 | grep "Server:"
 
+Upgrading to v1.4.0
```

Comment:
this looks great, by the way, but we probably need some more words about the fact that registration with email addresses won't work any more unless you twiddle some options? OTOH that sort of additional stuff should probably go in  separate PR so we can get this one landed.

Reply:
Have updated the text